tests/sentry/receivers/test_sentry_apps.py

```diff
@@ -340,6 +340,27 @@ def test_after_issue_assigned(self, delay):
             },
         )
 
+    def test_after_issue_reassigned(self, delay):
+        GroupAssignee.objects.assign(self.issue, self.assignee, self.user)
+
+        new_assignee = self.create_user(name="Berry", email="berry@example.com")
+        GroupAssignee.objects.assign(self.issue, new_assignee, self.user)
+
+        assert faux(delay).called_with(
+            installation_id=self.install.id,
+            issue_id=self.issue.id,
+            type="assigned",
+            user_id=self.user.id,
+            data={
+                "assignee": {
+                    "type": "user",
+                    "name": new_assignee.name,
+                    "email": new_assignee.email,
+                    "id": new_assignee.id,
```

Comment:
Check that the webhook was fired and the issue was assigned to `new_assignee`
(Before the change, this assertion was only true if the issue was assigned, then unassigned, then assigned again)